src/components/utils/popover.ts

```diff
@@ -434,7 +434,7 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
       el.dataset.itemName = item.name;
     }
     const label = Dom.make('div', Popover.CSS.itemLabel, {
-      innerHTML: item.label,
+      innerHTML: item.title,
```

Comment:
the title is optional, so we need to provide some fallback value

Reply:
done